cmd/snaptel/flags.go

```diff
@@ -77,8 +80,9 @@ var (
 		Usage: "The plugin name",
 	}
 	flPluginVersion = cli.IntFlag{
-		Name:  "plugin-version, v",
-		Usage: "The plugin version",
+		Name:   "plugin-version, v",
+		Usage:  "The plugin version",
+		EnvVar: "SNAP_PLUGIN_VERSION",
```

Comment:
This flag is used only in the following snaptel plugin commands:
- ` snaptel plugin swap <load_plugin_path> <unload_plugin_type>:<unload_plugin_name>:<unload_plugin_version>`
- ` snaptel plugin config get  <plugin_type>:<plugin_name>:<plugin_version>`
So it is used only to swapping plugins or get info about configuration requested by the plugin.

Based on those, could you tell me what is the use-case to declare the env var "SNAP_PLUGIN_VERSION"?